src/en/server-hosting/popular-forks-cdns.md

```diff
@@ -0,0 +1,39 @@
+# List of codebases with prebuilt server packages
+
+This document will serve as a list of forks known to have a prebuilt server software for players to download.
+
+This doc was last updated at 2024-10-01 13:52:04 UTC
+
+| Fork name | URL | Available platforms |
+|---|---|---|
+| Wizard's Den | [Link](https://wizards.cdn.spacestation14.com/fork/wizards) | Windows, macOS (x64/Intel), Linux (x64/ARM64) |
+| Harmony 14 | [Link](http://cdn.harmony14.com/fork/harmony/) | Windows, macOS (x64/Intel), Linux (x64/ARM64) |
+| Solaris 14 | [Link](https://ss14-cdn.mw-gc.com/fork/MWGSolaris14) | Windows, macOS (x64/Intel), Linux (x64/ARM64) |
+| Axolotl MRP | [Link](https://axolotl.yuniiworks.de/cdn/fork/axolotl/) | Linux (x64) |
+
+FAQ:
+
+- The fork I want is not listed here.
+	- You will need to contact the fork you want a server build for and ask them if they provide one. If they do suggest them to add themselves here.
+	
+- The fork I want is listed here, but my platform is not available for download.
+	- You will need to contact the fork asking if they are willing to build for your platform.
+	
+- The fork does not provide prebuilds (or any other issue where you are unable to retrive a server prebuild)
+	- Your only choice is to compile the forks codebase yourself if available. Instructions can be found [here](../general-development/setup/server-hosting-tutorial.md#level-2-server-with-custom-code)
+
+- I'm a server host and I want to add myself here.
+	- You are free to make a PR yourself in the docs or ping @vasilisiscool (322708417540259841) in #hosting on discord with all the information needed.
+
+<!-- 
```

Comment:
I think this should be visible and not in a comment?

Reply:
Thats not a comment

Reply:
It's not visible when rendered: https://github.com/space-wizards/docs/blob/892c9b9de6aae1649447a6f520086dcf37a4db71/src/en/server-hosting/popular-forks-cdns.md

Reply:
It is visible in mdbook

Reply:
I meant the rules, not the FAQ. The `<!--` comments it out so it doesn't render.

Reply:
I was confused since you reviewed a completely unrelated section. Most players don't care, so I commented it out. It's mostly for the reviewing maintainer. I guess its fine to include now that i think about it

Reply:
I reviewed the line that started the comment, sorry if it was unclear.